openvasreporting/libs/parser.py

```diff
@@ -58,7 +58,7 @@ def openvas_parser(input_files, min_level=Config.levels()["n"]):
         root = Et.parse(f_file).getroot()
 
         logging.debug("================================================================================")
-        logging.debug("= {}".format(root.find("./task/name").text))  # DEBUG
+#        logging.debug("= {}".format(root.find("./task/name").text))  # DEBUG
```

Comment:
Don't forget to revert this to prevent unnecessary changes

Reply:
agreed - can go back in but this syntax errors when running in Kali.. Hence the reason it was commented out